src/Pages/Login/login.scss

```
@@ -0,0 +1,153 @@
+@import '../../Styles/common.scss';
+@import '../../Styles/reset.scss';
+* {
+  box-sizing: border-box;
+}
+.login {
+  width: 40vw;
+  height: 100%;
+  margin: 0 auto;
+  .loginTitle {
+    width: 100%;
+    height: 30vh;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    h1 {
```

```suggestion
    align-items: center;
    
    h1 {
```
가독성을 위해 셀럭터 시작할 때는 한 줄 띄워주시기 바랍니다.